winui/AvatarView/visual-styles.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: post
+title: Visual Styles in WinUI AvatarView control | Syncfusion
+description: This section will explain about the different types of visual styles that exists in WinUI AvatarView.
```

Comment:
Description: This section will explain the different types of visual styles in WinUI AvatarView.